technical-docs/designs/repos-merge.md

```diff
@@ -0,0 +1,284 @@
+# Moving SPK to Bedrock Repo: A Phased Approach
+
+## Phase 1: Make Changes to Bedrock and SPK Repos
+
+### Changes to Bedrock Repo
+
+By making following changes in existing Bedrock repo minimizes the number of
+changes after the merge and limits the testing scope.
+
+#### Current Bedrock Repo
+
+```
+├── .github
+├── cluster
+├── docs
+├── gitops
+├── pipelines
+├── test
+├── tools
+├── LICENSE
+├── README.md
+├── .gitignore
+├── azure-pipelines.yml
+```
+
+#### Proposed Changes
+
+- Create `src` folder in `cluster` folder
+- Move all files and folders except `src` from `cluster` to `cluster\src`
+- Move `azure-pipelines.yml` file from the root to `cluster\build\pipelines`
+  directory and make sure all pipelines still works.
+- Move `test` folder to `cluster` folder and make sure all tests and pipelines
+  still works.
+- Move `bedrock` related files from `tools` directory to `cluster\build` folder
+  and make sure existing Bedrock release process still works.
+- [ ] Move following files form the `root` directory to `cluster` and make sure
+      `cluster` project can be opened in VS Code.
+  - LICENSE
+  - .gitignore
+
+#### Testing Changes
+
+After making the above changes, need to make sure that the follwing tests pass.
+
+- Verify the environments integration tests continue to work in `azdo`
+- Verify `bedrock` release provess work
+- Verify the project can be opened in VS Code from the `cluster` folder
+
+#### Bedrock repo after changes
+
+`Bedrock` repo would look like below after implementing the above changes.
+
+```
+├── .github
+├── gitops
+├── cluster
+    ├── build
+        ├── pipelines
+            ├── azure-pipelines.yml
+        ├── tools
+    ├── docs
+    ├── src
+    ├── test
+    ├── LICENSE
+    ├── README.md
+    ├── .gitignore
+
```

Comment:
from what i understand, we are turning on git pages feature. where is the docs folder has to be at the root.

Reply:
Agreed. I some how I missed this. Thanks @dennisseah  

@sarath-p for items in Phase 2 and beyond let's add a note that the  `docs` directory is reserved and we can re-visit